### PR TITLE
fix: sidetree-fabric well known domain name

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer1-org1.trustbloc.local.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "peer1-org1.trustbloc.local",
```
Comment (troyronda) [inline test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer1-org1.trustbloc.local.json:2]: I don’t understand. Is this some special test data?

There should be org1.trustbloc.local for the deployment.

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/testnet.trustbloc.local.json [hunk 2]
```diff
@@ -1,19 +0,0 @@
-{
-  "domain": "testnet.trustbloc.local",
```
Comment (troyronda) [inline test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/testnet.trustbloc.local.json:2]: This domain is the stakeholder domain. Why deleted?

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer0-org1.trustbloc.local.json [hunk 3]
```diff
@@ -4,9 +4,16 @@
   "policy": {
     "cache": {"max_age": 123456789}
   },
-  "endpoints": [
-    "https://peer0-org1.trustbloc.local/document",
-    "https://peer1-org1.trustbloc.local/document"
+  "members": [
```
Comment (troyronda) [inline test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer0-org1.trustbloc.local.json:7]: why a consortium file?

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer1-org1.trustbloc.local.json (hunk 1 above)
Comment (fqutishat) [reply test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer1-org1.trustbloc.local.json:2]: we don't have load balancer locally we use peer container name as domain

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/testnet.trustbloc.local.json (hunk 2 above)
Comment (fqutishat) [reply test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/testnet.trustbloc.local.json:2]: we don't have load balancer locally we use peer container name as domain

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer0-org1.trustbloc.local.json (hunk 3 above)
Comment (fqutishat) [reply test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer0-org1.trustbloc.local.json:7]: we don't have load balancer locally we use peer container name as domain

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer1-org1.trustbloc.local.json (hunk 1 above)
Comment (fqutishat) [reply test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer1-org1.trustbloc.local.json]: done

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/testnet.trustbloc.local.json (hunk 2 above)
Comment (fqutishat) [reply test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/testnet.trustbloc.local.json]: done

File: test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer0-org1.trustbloc.local.json (hunk 3 above)
Comment (fqutishat) [reply test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer0-org1.trustbloc.local.json]: done

**Pull request merged**